docs/source/science/HLS/HLSL30.ipynb

```diff
@@ -2,23 +2,87 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">creds = maap.aws.earthdata_s3_credentials('https://data.lpdaac.earthdatacloud.nasa.gov/s3credentials')
&nbsp; &nbsp; aws_session = boto3.session.Session(
&nbsp; &nbsp; &nbsp; &nbsp; aws_access_key_id=creds['accessKeyId'],&nbsp;
&nbsp; &nbsp; &nbsp; &nbsp; aws_secret_access_key=creds['secretAccessKey'],
&nbsp; &nbsp; &nbsp; &nbsp; aws_session_token=creds['sessionToken'],
&nbsp; &nbsp; &nbsp; &nbsp; region_name='us-west-2')
</pre><p>this is doing the same thing as the get_aws_session_DAAC func?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p><code>get_aws_session_DAAC</code> wraps the <code>boto3</code> session in an <code>rasterio.session.AWSSession</code> object. We could have it return the <code>boto3</code> session instead to avoid this duplicated code. </p>